ui/app/selectors/selectors.js

```diff
@@ -542,7 +543,8 @@ function getRenderablePermissionsDomains (state) {
         : ''
 
       return [ ...acc, {
-        name,
+        name: name ? name : domainKey,
+        origin: name ? domainKey : '',
```

Comment:
This seems misleading, and potentially dangerous. The origin should never be an empty string.
Maybe what this should be called is "subtitle" or "secondaryName" or something to reflect how it's used, and why you'd ever want it blank.

Reply:
Good suggestion, fixed in e0f889a0a